Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,5 +56,21 @@ public extension FileManager {
         return nil
     }
 
+    /// SwifterSwift: Save Encodable object to a given URL.
+    ///
+    /// - Parameters:
+    ///   - object: Object of type T that conforms to Encodable protocol.
+    ///   - url: URL to save object to.
+    ///   - encoder: JSONEncoder.
+    /// - Throws: Throws any errors thrown by JSONEncoder or FileManager.
+    public static func encode<T: Encodable>(_ object: T, to url: URL, using encoder: JSONEncoder=JSONEncoder()) throws {
+        do {
+            let encoder = encoder
+            let data = try encoder.encode(object)
+            FileManager.default.createFile(atPath: url.path, contents: data, attributes: nil)
```

Comment:
`createFile` returns a boolean value to indicate if it succeeded or failed. I think it would make sense to return it as the return value of this function so that users will be informed of success/failure.
In my project I have a custom `Error` that I `throw` if the call to `createFile` fails, however adding this isn't possible given the structure of SwifterSwift.